minclient/connection.go

```diff
@@ -360,7 +361,7 @@ func (c *connection) onTCPConn(conn net.Conn) {
 
 	// Allocate the session struct.
 	cfg := &wire.SessionConfig{
-		Geometry:          sphinx.DefaultGeometry(),
+		Geometry:          c.geo,
```

Comment:
Each component of katzenpost needs to have a Sphinx geometry, therefore it's natural to copy it around. We were doing that before, only from a hardcoded geometry.

The Packet API could be changed so that it reads a copy of the Sphinx geometry from elsewhere in the server's code.

SessionConfig needs to have the Sphinx geometry because some of the noise wire commands use the geometry to determine the payload size.